cookbook/sonoff_t1_us_3_gang.rst

```diff
@@ -0,0 +1 @@
+This is configuration for Sonoff T1 US 3 Gang : <https://github.com/genlink/esphome-docs/blob/patch-1/cookbook/sonoff_t1_us_3_gang.yaml>
```

Comment:
No, please edit `sonoff.rst` in the devices section instead. This file also won't show up in the index page because it hasn't been added there.